acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +84,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
```

Comment:
oh right.  it was because those are the only platforms supported by `dump_puppet_log`.

and indeed the only platforms supported by our replication here.  please retain this line